emulators/scummvm/Portfile

```diff
@@ -1,44 +1,133 @@
 PortSystem          1.0
 
 name                scummvm
-version             1.7.0
 platforms           darwin
 categories          emulators games
-maintainers         nomaintainer
+maintainers         {gmail.com:ken.cunningham.webuse @kencu} openmaintainer
 license             {GPL-2 BSD}
 description         cross-platform interpreter for several adventure engines
 long_description    \
     ScummVM is a cross-platform interpreter for several point-and-click \
-    adventure engines. This includes all SCUMM-based adventures by LucasArts, \
-    Simon the Sorcerer by AdventureSoft, and Beneath a Steel Sky and Broken \
-    Sword 2 by Revolution.
+    adventure engines. A current list of playable games is available at \
+    <https://www.scummvm.org/compatibility/>.
 
-homepage            http://scummvm.org/
-master_sites        sourceforge:project/${name}/${name}/${version}
-use_bzip2           yes
-checksums           sha256  d9ff0e8cf911afa466d5456d28fef692a17d47ddecfd428bf2fef591237c2e66 \
-                    rmd160  9c70db2cdc6631d6d8d6ac108c0cd58f1387e030
+homepage            https://scummvm.org/
+                    
+subport             ${name}-devel {}
+                                        
```

Comment:
Now interesting that port lint --nitpick didn't show me those. Good eyes! Removed.